formbricks/formbricks
# fix: adding loader for Delete and Save buttons

## What does this PR do?

This PR introduces a loader for the "Delete" and "Save" buttons in the DeleteDialog component. The loader provides visual feedback to users while the delete or save action is in progress.

Fixes #1338

Loom Video: https://www.loom.com/share/527e9519dc144b0d9ee425fa03b06d17?sid=c8fe270d-e1fd-40a6-ad71-0b51d62da012

## Type of change


## How should this be tested?

- Open the DeleteDialog component.
- Click the "Delete" button.
- Verify that the "Deleting..." loader is displayed.
- Ensure that the "Delete" button remains clickable during the loading state.
- Repeat the above steps for the "Save" button.

## Checklist

### Required

- [] Ran `pnpm build`

### Appreciated
